test/testLoadData.jl

```diff
@@ -0,0 +1,97 @@
+using GigaSOM, DataFrames, XLSX, CSV, Test, Random, Distributed, SHA, JSON
```

Comment:
can this test be combined with the `testIO.jl` file?

Reply:
I would prefer to keep this test separated as testFileIO is using the "classic" approach and will be depreciated in a later stage.

Reply:
ok with me